inlong-common/src/main/java/org/apache/inlong/common/pojo/sort/SortConfigResponse.java

```diff
@@ -36,6 +36,6 @@ public class SortConfigResponse {
     String msg;
     int code;
     String md5;
-    String data;
+    byte[] data;
```

Comment:
HTTP request need to string data.
byte array can be transformed to string by Base64